package.json

```diff
@@ -5,6 +5,7 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "@popmotion/easing": "^1.0.2",
```

Comment:
Yes, with the `import linter` finds dependencies in use that are not on the package.json and it reports them. `lodash` and `@popmotion` were some of the culprits. Importing these dependencies work because other dependencies on the project already use them.